src/parcel.F

```diff
@@ -1221,8 +1221,6 @@ subroutine parcel_interp(dt,mtime,xh,uh,ruh,xf,uf,yh,vh,rvh,yf,vf, &
     nploop2:  &
     DO np=1,nparcelsLocal
 
-      pdata(np,prtime) = mtime
```

Comment:
I wanted each particle to know how long it have lived (i.e. stored in pdata). But then I realized that there was already a pdata "prtime" that wasn't being used, so I just repurposed it. It used to simply record the model time, but now I changed it so that it provides total accumulated time since the particle was created.

Reply:
thanks for the clarification.